### PR TITLE
v6:Turret clapper

### PR DESCRIPTION
**Pull request by SnakkiZXZ**
After looking at the new units, I decided to make a turret with Quad shells. This turret is designed as an improved version of ripple using plastanium and surge alloy.
Screenshot_20200927_154114
Screenshot_20200927_154056

`Criticism and corrections are welcome, it is possible that it is a little unbalanced, but this is solved in a couple of seconds by changing the parameters of both the turret and the bullet.`

### CODE REVIEW

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1082,6 +1082,7 @@ block.wave.name = Wave
 block.swarmer.name = Swarmer
 block.salvo.name = Salvo
 block.ripple.name = Ripple
+block.clapper.name = Clapper
```
Comment (l-Luna) [inline core/assets/bundles/bundle.properties:1085]: kind of an odd name

File: core/assets/bundles/bundle.properties [hunk 2]
```diff
@@ -1303,6 +1304,7 @@ block.swarmer.description = A medium-sized missile turret. Attacks both air and
 block.salvo.description = A larger, more advanced version of the Duo turret. Fires quick salvos of bullets at the enemy.
 block.fuse.description = A large, close-range shrapnel turret. Fires three piercing blasts at nearby enemies.
 block.ripple.description = An extremely powerful artillery turret. Shoots clusters of shells at enemies over long distances.
+block.clapper.description = A powerful turret, shoots large plastane bombs, handy against large clusters of weak enemies.
```
Comment (l-Luna) [inline core/assets/bundles/bundle.properties:1307]: Do you mean "...large plastanium bombs; handy against..."?

File: core/src/mindustry/content/Blocks.java [hunk 3]
```diff
@@ -1527,6 +1527,24 @@ public void drawBase(Tile tile){}
 
             consumes.powerCond(3f, (TractorBeamBuild e) -> e.target != null);
         }};
+        
+        clapper = new ItemTurret("clapper"){{
+            requirements(Category.turret, with(Items.plastanium, 230, Items.graphite, 150, Items.surgealloy, 130, Items.metaglass, 70));
+            ammo(
+            Items.graphite, plastBomb
```
Comment (l-Luna) [inline core/src/mindustry/content/Blocks.java:1534]: If it creates "plastanium bombs", shouldn't it [have the option to] consume plastanium?

Comment (SnakkiZXZ) [reply core/src/mindustry/content/Blocks.java:1534]: Oh, sorry, I just completely forgot to change the shell

File: core/assets/bundles/bundle.properties (hunk 2 above)
Comment (SnakkiZXZ) [reply core/assets/bundles/bundle.properties:1307]: sorry, my knowledge of English is limited to google translator

File: core/src/mindustry/content/Blocks.java [hunk 4]
```diff
@@ -1527,6 +1527,22 @@ public void drawBase(Tile tile){}
...
+            targetAir = false;
+            health = 2400;
+            size = 4;
+            inaccuracy = 1.1f;
+            reloadTime = 120f;
+            shots: 1;
+            range: 200f;
+            recoilAmount: 7f;
+            cooldown: 0.01f;
+            shootShake: 5f;
```
Comment (summetdev) [inline core/src/mindustry/content/Blocks.java:1543]: This doesn't compile. Test changes before submitting them.

**Comment by TheSlaylord** [comment]
`Qude`, `plastane`, `kinetic alloy`
So much pain in a single line ^

**Comment by SnakkiZXZ** [comment]
@TheSlayLord it's okay now?
I removed the extra lines a little, I hope now everything works

**Review by genNAowl — CHANGES_REQUESTED** [review]
Should fix compilation error without breaking turret?

**Comment by sk7725** [comment]
image
What is this? hmm

**Comment by SnakkiZXZ** [comment]
@sk7725 This is from a mod, I test items in the form of a mod before compiling into the game. Specifically, this is a mod source

**Comment by SnakkiZXZ** [comment]
I thought for a long time and looked for a suitable effect for firing this turret. I created a new `bigPlastShoot`
Screenshot_20200928_150627
Screenshot_20200928_150614

 effect and removed the smoke effect. Also thanks for helping me to notice and fix errors.

**Comment by sk7725** [comment]
> @sk7725 This is from a mod, I test items in the form of a mod before compiling into the game. Specifically, this is a mod source

you got me interested, what is the difference of that and a vanilla item source?

**Comment by SnakkiZXZ** [comment]
@sk7725 everything is simple, it has more health: 6000. and a more beautiful texture) nothing more

**Comment by sk7725** [comment]
sad, I thought it was an auto-switching source

**Comment by SnakkiZXZ** [comment]
@sk7725 you interested me, I can create it like a copper reenactor, which as a source can summon any unit and even change your team. Switching source would be a great addition to my mod.

File: core/assets/bundles/bundle.properties (hunk 1 above)
Comment (SnakkiZXZ) [reply core/assets/bundles/bundle.properties:1085]: initially the turret was called a ```plastanium cracker```, then I decided to make the name more fluoristic and called ```xenolit```, but in the end I returned to the old name and decided to leave it just a ```clapper```

**Comment by Anuken** [comment]
I may reconsider this in the future, but right now there isn't much of a niche for an upgraded ripple. The Foreshadow turret can do long-range attacks, and ripple turrets can still do medium-range artillery.

**Pull request closed without merging**